### DISCUSSION TITLE
Boilerplate link not working

### DISCUSSION THREAD

**Discussion by FridaSvanberg** [DevSkillsHQ/discusssions#77] [Help]
When I use the link from the readme file [https://docs.devskills.co/collections/85-the-interview-process/articles/342-importing-challenge-boilerplate](https://docs.devskills.co/collections/85-the-interview-process/articles/342-importing-challenge-boilerplate) I just get a 404
<img width="600" alt="Screenshot 2024-03-21 at 15 03 24" src="https://github.com/DevSkillsHQ/discusssions/assets/38505160/1cbba69d-97a7-4b47-a686-3bffd34544a9">

Is there something wrong with the URL or does it just not exist anymore?

**Comment by ehsan-alva** [comment]
Hello @FridaSvanberg, this is the correct link to use https://help.alvalabs.io/en/articles/7972852-supported-coding-test-boilerplates we have also updated your challenge README, sorry for the inconvenience.
